spec/yext/api/knowledge_api/knowledge_manager/location_spec.rb

```diff
@@ -7,13 +7,28 @@
   it_behaves_like "default scopes"
 
   describe "index" do
-    it_behaves_like "sets rate limits", -> { Yext::Api::KnowledgeApi::KnowledgeManager::Location.all.to_a }
+    it_behaves_like "sets rate limits", -> { Yext::Api::KnowledgeApi::KnowledgeManager::Location.account("my-test-account-id-2").all.to_a }
```

Comment:
I wasn't testing Locations properly before due to my not understanding them properly.

So, I fixed this test.